google/cloud/aiplatform/models.py

```diff
@@ -604,25 +622,25 @@ def _validate_deploy_args(
             raise ValueError("Max replica cannot be negative.")
         if deployed_model_display_name is not None:
             utils.validate_display_name(deployed_model_display_name)
-
-        if traffic_split is None:
+        if traffic_percentage and traffic_split:
```

Comment:
Fixed, removed the line checking for both and explicitly set the values of `traffic_percentage` and `traffic_split` instead of exposing the arguments.